frontend/sw360-portlet/src/main/webapp/html/components/view.jsp

```diff
@@ -298,7 +295,6 @@
                         {"title": "Component Type", data: "cType"},
                         {"title": "Actions", data: "id", render: {display: renderComponentActions}}
                     ];
-                    order = [[2, 'asc']];
                     columnDefs = [{ "orderable": false, "targets": [0, 4, 5, 6 ,7] }];
```

Comment:
This line should be removed as well, since its declaration was removed and it is not used anymore

Reply:
was my bad, `columnDefs` should not be removed... only `order`

Reply:
Are you sure? I did not found any other reference to columnDefs.